Source/ACE/Entity/Creature.cs

```diff
@@ -115,6 +128,54 @@ private void SetAbilities(AceCreatureObject aco)
             abilities.Add(Enum.Ability.Mana, Mana);
 
             Abilities = new ReadOnlyDictionary<Enum.Ability, CreatureAbility>(abilities);
+
+            IsAlive = true;
+        }
+
+        public void Kill(Session session)
+        {
+            IsAlive = false;
+
+            // Send UpdateHealth Message with 0 Health - this is not necessary according to live pcaps
+            // var healthLossEvent = new GameEventUpdateHealth(session, Guid.Full, 0f);
+            // session.Network.EnqueueSend(healthLossEvent);
+
+            // 1. step in retail: Create and send the death notice
+            string killMessage = $"{session.Player.Name} has killed {Name}.";
+            var creatureDeathEvent = new GameEventDeathNotice(session, killMessage);
+            session.Network.EnqueueSend(creatureDeathEvent);
+
+            // 2. step in retail: SoundEvent: sound 48 (HitFlesh1), volume 0,5
+            session.Player.ActionApplySoundEffect(Sound.HitFlesh1, Guid);
```

Comment:
In the pcap I was analyzing that sound effect had the objectID of the drudge. But I am with you that this has to come from executing a combat action.